website/integrations/services/mautic/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Integrate with Mautic
+sidebar_label: Mautic
+support_level: community
+---
+
+## What is Mautic
+
+> Mautic provides free and open source marketing automation software available to everyone. Free email marketing and lead management software.
+>
+> -- https://mautic.org/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `mautic.company` is the FQDN of the Mautic installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::info
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+### Prepare the certificates
+
+Mautic and authentik both require an X.509 certificates.
+Mautic requires the key to be in PKCS#1-format specifically (and thus RSA).
+
+To avoid changing certificates in authentik (e.g., if they are present in PKCS#8), we generate a new one:
```

Comment:
```suggestion
To avoid changing certificates in authentik (e.g., if they are present in PKCS#8), go to the authentik Admin interface and generate a new one:
```